pkg/network/ovs_test.go

```diff
@@ -0,0 +1,49 @@
+package network
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	opv1alpha1 "github.com/kubevirt/cluster-network-addons-operator/pkg/apis/networkaddonsoperator/v1alpha1"
+)
+
+var _ = Describe("Testing Ovs", func() {
```

Comment:
Why do we have two describe here ?

Reply:
One if the for component we test and the other one is for the function we test

Reply:
Fair enoough